### PR TITLE
MM, Metadb derived table Instance_ext changed Date to timestamptz

### PR DESCRIPTION
**Pull request by jenmawe**
For system created FOLIO fields of cataloged and updated date change casting to timestamptz.

### CODE REVIEW

**Review by lib-eric — APPROVED** [review]
Small change -- thanks!

**Review by adoerrer — CHANGES_REQUESTED** [review]
Thanks you for changing the date type to the more accurate timestampz. The query runs and looks goods, but I'd suggest to create indexes for `title` and `index_title` as well.

- [x] Query runs
- [x] Data output makes sense
- [x] Data types look correct
- [x] Query logic seems correct
- [x] Column names make sense and are spelled correctly
- [x] Query syntax and style are appropriate (e.g., correct use of double quotes, lower case for data types)

For derived tables:
- [] query has indexes on columns
- [x] vacuum analyze has been added to the bottom
- [x] table appears in runlist.txt at an appropriate point

**Comment by nassibnassar** [comment]
@adoerrer @jenmawe I think `title` and `index_title` may tend to be large varchars and unsuitable for the default index type.  They should probably be left without an index for now.

**Comment by jenmawe** [comment]
> @adoerrer @jenmawe I think `title` and `index_title` may tend to be large varchars and unsuitable for the default index type. They should probably be left without an index for now.

Both can get really long. Let me remove these from the index.

**Review by adoerrer — APPROVED** [review]
Oh, I didn't know  `title` and `index_title` can get that big. Since they are not created atm @jenmawe you don't have to do anything. 
Thanks, for this work!

File: sql_metadb/derived_tables/instance_ext.sql [hunk 1]
```diff
@@ -22,8 +22,8 @@ SELECT
     jsonb_extract_path_text(i.jsonb, 'modeOfIssuanceId')::uuid AS mode_of_issuance_id,
     moit.name AS mode_of_issuance_name,
     jsonb_extract_path_text(i.jsonb, 'catalogedDate')::date AS cataloged_date,
-    jsonb_extract_path_text(i.jsonb, 'metadata', 'createdDate')::date AS created_date,
-    jsonb_extract_path_text(i.jsonb, 'metadata', 'updatedDate')::date AS updated_date
+    jsonb_extract_path_text(i.jsonb, 'metadata', 'createdDate')::timestamptz AS created_date,
```
Comment (nassibnassar) [inline sql_metadb/derived_tables/instance_ext.sql:25]: The `createdDate` data do not appear to have a timezone.  It should probably have the `timestamp` data type.

**Review by nassibnassar — CHANGES_REQUESTED** [review]

**Comment by nassibnassar** [comment]
It looks like you changed `updatedDate` instead of `createdDate`.

**Comment by jenmawe** [comment]
> It looks like you changed `updatedDate` instead of `createdDate`.

Sorry about that. I just updated the createdDate. This should be good to go.

**Comment by nassibnassar** [comment]
Almost!  The `updatedDate` field should be changed back to `timestamptz`.

**Comment by jenmawe** [comment]
> Almost! The `updatedDate` field should be changed back to `timestamptz`.

Changed it back.

**Pull request merged**